src/assets/scripts/client/airport/AirportController.js

```diff
@@ -191,6 +191,13 @@ class AirportController {
         return this.airports[icao.toLowerCase()];
     }
 
+    getTransAlt() {
+        if (Object.prototype.hasOwnProperty.call(this.current, 'transitionAltitude')) {
```

Comment:
- we tend to use lodash `_has` (https://lodash.com/docs/4.17.15#has) for these sorts of checks.
- could you flip the logic here to return early when `transistionAltitude` is **not** set?